src/main/java/org/fundacionjala/coding/rodrigo/movierefactor/Movie.java

```diff
@@ -0,0 +1,60 @@
+package org.fundacionjala.coding.rodrigo.movierefactor;
+
+/**
+ * Abstract class for movie.
+ */
+public abstract class Movie {
+    private String title;
+    private int daysRented;
+    protected double charge;
+
+    /**
+     * Constructor.
+     *
+     * @param title      of the movie.
+     * @param daysRented of the movie.
+     */
+    Movie(final String title, final int daysRented) {
+        this.title = title;
+        this.daysRented = daysRented;
+    }
+
+    /**
+     * Some calc of the charge.
+     */
+    abstract void calculateCharge();
```

Comment:
this abstract method should return double.
So the getCharge method will be removed.

Reply:
done